cranelift/codegen/src/ir/dfg.rs

```diff
@@ -307,7 +328,7 @@ impl DataFlowGraph {
     /// For each argument of inst which is defined by an alias, replace the
     /// alias with the aliased value.
     pub fn resolve_aliases_in_arguments(&mut self, inst: Inst) {
-        for arg in self.insts[inst].arguments_mut(&mut self.value_lists) {
+        for arg in self.insts.0[inst].arguments_mut(&mut self.value_lists) {
```

Comment:
I suppose this case can use the `IndexMut` implementation on the newtype wrapper, instead of needing to dig inside it. I guess I have a mild preference for using the public API where that's reasonable.

Reply:
I was just digging into this, and found that this function handles cases where the arguments aren't contained in an entity list as well.

Reply:
I think I had a recency bias issue and thought you were referring to refactoring `arguments_mut` instead of removing the unnecessary use of `.0`. I'll agree that this is better without the `.0` :+1: